gradle.properties

```diff
@@ -97,8 +97,8 @@ kotlinWasmEnabled=false
 # To know which version, should be used, see compose/frameworks/support/libraryversions.toml
 oel.publication=true
 # Look for `COMPOSE` in libraryversions.toml
-oel.androidx.version=1.5.0-beta03
+oel.androidx.version=1.5.0
 # Look for `COMPOSE_MATERIAL3` in libraryversions.toml
 oel.androidx.material3.version=1.1.1
-oel.androidx.foundation.version=1.5.0-beta03
-oel.androidx.material.version=1.5.0-beta03
+oel.androidx.foundation.version=1.5.0
```

Comment:
Merge after https://github.com/JetBrains/compose-multiplatform-core/pull/743/files

Reply:
Maybe we need 1.5.0-rc01 first?